openfeature-provider/go/confidence/swap_wasm_resolver_api_test.go

```diff
@@ -123,29 +257,27 @@ func TestSwapWasmResolverApi_WithRealState(t *testing.T) {
 	}
 	defer swap.Close(ctx)
 
-	// Resolve the tutorial-feature flag using the real client secret from the state
-	// The state includes client secret: mkjJruAATQWjeY7foFIWfVAcBWnci2YF
-	// Use "tutorial_visitor" as the visitor_id to match the segment targeting
-	request := &resolver.ResolveFlagsRequest{
-		Flags:        []string{"flags/tutorial-feature"},
-		Apply:        false,
-		ClientSecret: "mkjJruAATQWjeY7foFIWfVAcBWnci2YF",
-		EvaluationContext: &structpb.Struct{
-			Fields: map[string]*structpb.Value{
-				"visitor_id": structpb.NewStringValue("tutorial_visitor"),
-			},
-		},
-	}
+	request := createResolveWithStickyRequest(
+		createTutorialFeatureRequest(),
+		nil,   // empty materializations
+		false, // failFast
```

Comment:
Shouldn't we keep this as our implemention?
```suggestion
		true, // failFast
```